### ISSUE TITLE
App crashes when connecting to a RC server with URL contains subdirectory.

### ISSUE THREAD

**Issue by ura14h** [RocketChat/Rocket.Chat.ReactNative#1605]
- Your Rocket.Chat Experimental app version: 4.2.4
- Your Rocket.Chat server version: 2.4.1
- Device (or Simulator) you're running with: iOS

App crashes when connecting to a RC server with URL contains subdirectory. (e.g. `https://www.example.com/apps/chat`)

I guess that `DB.setShareDB()` and `DB.setActiveDB()` in app/lib/database/index.js fails to replace URL to a path correcty in this case.

I haven't built and tested App, but I think it's better to add a global flag to the regular expression pattern parameter of replace function.

For example:
from `const path = database.replace(/(^\w+:|^)\/\//, '').replace(/\//, '.');`
to `const path = database.replace(/(^\w+:|^)\/\//, '').replace(/\//g, '.');`

Related old issue #1322

Thank you.

**Comment by djorkaeffalexandre** [comment]
Hello @ura14h!
Thanks a lot about your suggestion, is correct.
Can you open a Pull Request with these changes?

**Comment by ura14h** [comment]
I can't do that.

I can only change the source code, but I can't verify that it works correctly with on the current master version.

First, I setted up my environment according to the "Installing Dependencies" section of the README. I ran the current master version, but the iOS simulator showed the error "Could not connect to server".
In this situation, I cannot confirm that my patch is correct.

I am using Xcode version 11.3.1.
